add apps debug section to troubleshooting guide (#7202)
PostHog/posthog.com: contents/docs/product-analytics/troubleshooting.mdx

There are a few common reasons that you may not see events appear in your project:

> ### 1. Your library is configured incorrectly.

Reviewer: We dont need "." in headings (not sure if 
```suggestion
### 1. Your library is configured incorrectly
```

![Events visible on webhook.site](../../images/docs/product-analytics/events-in-webhook.site.png)

> ### 2. An app has been configured incorrectly.

Reviewer: ```suggestion
### 2. An app has been configured incorrectly
```

> Another common reason for missing events is that certain plugins have been configured to drop certain or all events. To debug events follow these steps in the following order:

Reviewer: ```suggestion
Another common reason for missing events is that certain plugins have been configured to drop certain or all events. To debug if this is the case, follow the below steps in order:
```

> 1. Make sure the Filter Out plugin is correctly configured.

Reviewer: ```suggestion
1. Make sure the [Filter Out](/cdp/filter-out) plugin is correctly configured.
```

2. Try to temporarily disable the Filter Out plugin.

> 3. Try to temporarily disable all of the other enabled plugins.

Reviewer: ```suggestion
3. Try to temporarily disable all other enabled plugins.
```

> ### 3. There is a bug the library.

Reviewer: ```suggestion
### 3. There is a bug the library=
```

![network request being sent to posthog](../../images/docs/product-analytics/network-request-to-posthog.png)

> ### 4. Events are not ingested after being received by PostHog.

Reviewer: ```suggestion
### 4. Events are not ingested after being received by PostHog
```
